autogpt/commands/news_aggregation.py

```diff
@@ -0,0 +1,46 @@
+import concurrent.futures
+
+from newsapi import NewsApiClient
+
+from autogpt.commands.command import command
+from autogpt.config import Config
+
+CFG = Config()
+
+categories = ["technology", "business", "entertainment", "health", "sports", "science"]
+news_api_client = NewsApiClient(CFG.news_api_key)
+
+
+def get_news_headlines_for_category(category: str):
```

Comment:
type hint for return type?

Reply:
done

Reply:
I mean, it should be `def get_news_headlines_for_category(category: str) -> list[str]:`